firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -221,32 +222,36 @@ void showFeedbackNotification(
    * Displays a notification that, when tapped, will take a screenshot of the current activity, then
    * start a new activity to collect and submit feedback from the tester along with the screenshot.
    *
-   * <p>On Android 13 and above, this method requires the <a
-   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
-   * permission for sending notifications</a>: {@code POST_NOTIFICATIONS}. If your app targets
-   * Android 13 (API level 33) or above, you should <a
+   * <p>On Android 13 and above, this method requires the runtime permission for sending
+   * notifications: <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">{@code
```

Comment:
I'm not sure if `{@code …}` inside `<a>…</a>` gets rendered properly. Consider using `<pre>…</pre>` instead.

Reply:
Ditto